File: src/main/kotlin/tech/diggle/apps/accomodation/candidates/CandidateService.java
package tech.diggle.apps.accomodation.candidates;

import java.util.List;

public interface CandidateService {
    Candidate findCandidateByCandidateName(String candidateName);

    Candidate findCandidateByParty(String party);

    Candidate updateCandidate(Candidate candidate);

    Candidate createCandidate(Candidate candidate);

    boolean deleteCandidate(Candidate candidate);

    List<Candidate> listAll();
}

File: src/main/kotlin/tech/diggle/apps/accomodation/candidate/CandidateService.kt
package tech.diggle.apps.accomodation.candidate

interface CandidateService {
    fun getAll(): List<Candidate>
    fun getAll(id: Long): List<Candidate>
    fun add(candidate: Candidate): Candidate
    fun delete(id: Long)
    fun find(candidate: Candidate):Candidate?
}